src/tool_cfgable.h

```diff
@@ -177,7 +177,8 @@ struct OperationConfig {
   bool doh_verifystatus;
   bool create_dirs;
   bool ftp_create_dirs;
-  bool ftp_skip_ip;
+  long ftp_pasvp_ip_rule;   /* how to handle the IP address the FTP server
+                               passes on to us */
```

Comment:
I think you should keep treating this as a boolean cmdline option

Reply:
original author permits 3 values, so I don't think bool would fit here.